doc/build/changelog/whatsnew_20.rst

```diff
@@ -751,7 +751,7 @@ The new ORM Declarative features introduced above at
 :ref:`whatsnew_20_orm_declarative_typing` introduced the
 new :func:`_orm.mapped_column` construct and illustrated type-centric
 mapping with optional use of :pep:`593` ``Annotated``.  We can take
-the mapping one step further by integrating with with Python
+the mapping one step further by integrating this with Python
```

Comment:
I'm not a native English speaker, but it sounds better without this. What do you think @zzzeek ?

Reply:
i think its fine!